### PR TITLE
Add roles and permissions to the document

### PR DESCRIPTION
**Pull request by WARHORSE-PRO**

### CODE REVIEW

File: features.md [hunk 1]
```diff
@@ -1,17 +1 @@
...
-
-## Subscribe for latest Job Post
-
-- Component consisting of a subscribe button.On clicking the subscribe button the user will be subscribed to all the latest Job posts by the organisation via email notifications. [feature link](https://github.com/fytinnovations/oc-careers/issues/1)
-
-## Schedule an interview or Reject the Applicant through the Applications Module
-
-- Schedule an Interview with the applicant or reject the Application from the Applications Module itself. [feature link](https://github.com/fytinnovations/oc-careers/issues/2)
-
-> Request for new features by creating an Issue on the [repository](https://github.com/fytinnovations/oc-careers/). Don't Forget to attach the "New Feature" label to it !
```
Comment (4nik3t) [inline features.md:17]: Why was this section removed

File: getting-started/permissions.md [hunk 2]
```diff
@@ -2,12 +2,37 @@
...
 
-## Assign Tickets
+## Permissions
 
- Support staff with this permission have the ability to assign the tickets to other support staff 
+Helpdesk has a very simple set of permissions namely **Manage HelpDesk**, **Assign Tickets**, **Self Assign Ticket**, **Create Tickets**.
+
+1. **Manage Helpdesk**
+
+    user can manage the entire helpdesk.
```
Comment (4nik3t) [inline getting-started/permissions.md:17]: Why is this not capitalized

File: features.md (hunk 1 above)
Comment (WARHORSE-PRO) [reply features.md:17]: Not relevant know it was just a copy of oc-careers

File: getting-started/permissions.md (hunk 2 above)
Comment (WARHORSE-PRO) [reply getting-started/permissions.md:17]: This is just a shoot point and not a sentence . I have done that purposely

**Pull request merged**